tango_simlib/utilities/base_parser.py

```diff
@@ -1,3 +1,6 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
```

Comment:
Please move this after the copyright block, to be consistent with the other modules - we always have the imports after the initial docstring.